### PR TITLE
[WIP DRAFT] Adds few clothing items.

### PR DESCRIPTION
**Pull request by Smol42**
to be fixed

### CODE REVIEW

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 1]
```diff
@@ -165,6 +165,55 @@
 /datum/greyscale_config/standalone_hood/worn/oldvox
 	icon_file = 'modular_skyrat/modules/GAGS/icons/head/head_oldvox.dmi'
 
+// MAID HEADWEAR
+
+/datum/greyscale_config/maid_headgear
+	icon_file = 'modular_skyrat/modules/GAGS/icons/maid_costume.dmi'
+	name = "Maid headgear"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Maid Headgear"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 2]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/maid_costume/maid_headgear.json'
+
+/datum/greyscale_config/maid_headgear/worn
+	name = "Maid headgear (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Maid Headgear (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 3]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/maid_costume/maid_headgear_worn.json'
+
+// BOWS
+
+/datum/greyscale_config/back_bow
+	icon_file = 'modular_skyrat/modules/GAGS/icons/bows.dmi'
+	name = "Back bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Back Bow"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 4]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/back_bow.json'
+
+/datum/greyscale_config/back_bow/worn
+	name = "Back bow (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Back Bow (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 5]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/back_bow_worn.json'
+
+/datum/greyscale_config/large_bow
+	icon_file = 'modular_skyrat/modules/GAGS/icons/bows.dmi'
+	name = "Large bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Large Bow"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 6]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/large_bow.json'
+
+/datum/greyscale_config/large_bow/worn
+	name = "Large bow (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Large Bow (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 7]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/large_bow_worn.json'
+
+/datum/greyscale_config/small_bow
+	icon_file = 'modular_skyrat/modules/GAGS/icons/bows.dmi'
+	name = "Small bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Small Bow"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 8]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/small_bow.json'
+
+/datum/greyscale_config/small_bow/worn
+	name = "Small bow (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Small Bow (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 9]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/small_bow_worn.json'
+
+/datum/greyscale_config/sweet_bow
+	icon_file = 'modular_skyrat/modules/GAGS/icons/bows.dmi'
+	name = "Sweet bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Sweet Bow"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 10]
```diff
@@ -165,6 +165,55 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/bows/sweet_bow.json'
+
+/datum/greyscale_config/sweet_bow/worn
+	name = "Sweet bow (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Sweet Bow (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 11]
```diff
@@ -236,6 +285,17 @@
 	name = "Colourable Sterile Mask (Worn) (Teshari)"
 	icon_file = 'modular_skyrat/modules/GAGS/icons/masks_teshari.dmi'
 
+//FACE SCARF
+
+/datum/greyscale_config/face_scarf
+	icon_file = 'modular_skyrat/modules/GAGS/icons/masks.dmi'
+	name = "Face scarf"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Face Scarf"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 12]
```diff
@@ -236,6 +285,17 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/face_scarf/face_scarf.json'
+
+/datum/greyscale_config/face_scarf/worn
+	name = "Face scarf (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Face Scarf (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 13]
```diff
@@ -288,6 +348,44 @@
 	name = "Shroud (Worn)"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/cloak/shroud_worn.json'
 
+/datum/greyscale_config/cloak/wide_cape
+	name = "Wide cape"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Wide Cape"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 14]
```diff
@@ -288,6 +348,44 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/cloak/wide_cape.json'
+	icon_file = 'modular_skyrat/modules/GAGS/icons/cloak.dmi'
+
+/datum/greyscale_config/cloak/wide_cape/worn
+	name = "Wide cape (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Wide Cape (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 15]
```diff
@@ -288,6 +348,44 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/cloak/wide_cape_worn.json'
+
+/datum/greyscale_config/cloak/robes
+	icon_file = 'modular_skyrat/modules/GAGS/icons/cloak.dmi'
+	name = "Robes"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Robe Cape"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 16]
```diff
@@ -288,6 +348,44 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/cloak/robes.json'
+
+/datum/greyscale_config/cloak/robes/worn
+	name = "Robes (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Robe Cape (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 17]
```diff
@@ -288,6 +348,44 @@
...
+
+/datum/greyscale_config/cloak/cape/worn
+	name = "Cape (Worn)"
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/cloak/wide_cape_worn.json'
+
+// MAID NECKWEAR
+
+/datum/greyscale_config/maid_costume/maid_collar
+	icon_file = 'modular_skyrat/modules/GAGS/icons/maid_costume.dmi'
+	name = "Maid collar"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Maid Collar"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 18]
```diff
@@ -288,6 +348,44 @@
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/maid_costume/maid_collar.json'
+
+/datum/greyscale_config/maid_costume/maid_collar/worn
+	name = "Maid collar (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
	name = "Maid Collar (Worn)"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 19]
```diff
@@ -1056,41 +1210,18 @@ digi
...
-/datum/greyscale_config/clothwraps/worn
+/datum/greyscale_config/wraps/worn
 	name = "Cloth Wraps (Worn)"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/wraps/wraps_worn.json'
 
-/datum/greyscale_config/clothwraps/worn/digi
+/datum/greyscale_config/wraps/worn/digi
 	name = "Cloth Wraps (Worn, Digi)"
 
-/datum/greyscale_config/legwraps
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:1071]: Pls put these back!

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 20]
```diff
@@ -1298,3 +1429,16 @@ digi
...
 	name = "Gauntlets (Worn)"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/primitive_catgirl_clothing/gauntlets_worn.json'
+
+/*
+*	GLOVES
+*/
+
+/datum/greyscale_config/maid_gloves
+	icon_file = 'modular_skyrat/modules/GAGS/icons/maid_costume.dmi'
+	name = "Maid gloves"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:1439]:
```suggestion
	name = "Maid Gloves"
```

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 21]
```diff
@@ -1298,3 +1429,16 @@ digi
...
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/maid_costume/maid_gloves.json'
+
+/datum/greyscale_config/maid_gloves/worn
+	name = "Maid gloves (Worn)"
```
Comment (Nerev4r) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:1443]:
```suggestion
	name = "Maid Gloves (Worn)"
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 22]
```diff
@@ -64,15 +64,6 @@
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
 
-/obj/item/clothing/head/hats/sec/cap/peacekeeper/sol/dark
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:67]: Also this.

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 23]
```diff
@@ -89,16 +80,6 @@
 	icon_state = "policetrafficcap"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
 	armor_type = /datum/armor/head_helmet
-/obj/item/clothing/head/helmet/sec/sol
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:92]: Also this.

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 24]
```diff
@@ -250,3 +231,45 @@
 	icon_state = "cowboyhat_black"
 	inhand_icon_state = "cowboy_hat_black"
 	supports_variations_flags = CLOTHING_SNOUTED_VARIATION_NO_NEW_ICON
+
+//Head bows - Sprites by @supugaru
+
+/obj/item/clothing/head/small_bow
+	name = "Small bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:238]:
```suggestion
	name = "small bow"
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 25]
```diff
@@ -250,3 +231,45 @@
...
+	greyscale_config_worn = /datum/greyscale_config/small_bow/worn
+	greyscale_colors = "#ffffff"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/head/small_bow/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/toggle_clothes, "gags_small_bow_t")
+
+/obj/item/clothing/head/large_bow
+	name = "Large bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:251]:
```suggestion
	name = "large bow"
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 26]
```diff
@@ -250,3 +231,45 @@
...
+	desc = "A large bow that you can place on the top of your head."
+	icon_state = "gags_large_bow"
+	greyscale_config = /datum/greyscale_config/large_bow
+	greyscale_config_worn = /datum/greyscale_config/large_bow/worn
+	greyscale_colors = "#ffffff"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/head/back_bow
+	name = "Back bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:260]:
```suggestion
	name = "back bow"
```

File: modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm [hunk 27]
```diff
@@ -86,3 +86,30 @@
 	greyscale_config_worn_better_vox = /datum/greyscale_config/mantle/worn/newvox
 	greyscale_config_worn_vox = /datum/greyscale_config/mantle/worn/oldvox
 	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/neck/maid_collar
+	name = "Custom maid neck cover"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm:91]:
```suggestion
	name = "maid neck cover"
```

File: modular_skyrat/modules/customization/modules/clothing/head/head.dm [hunk 28]
```diff
@@ -250,3 +231,45 @@
...
+	desc = "A large bow that you can place on the back of your head."
+	icon_state = "gags_back_bow"
+	greyscale_config = /datum/greyscale_config/back_bow
+	greyscale_config_worn = /datum/greyscale_config/back_bow/worn
+	greyscale_colors = "#ffffff"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/head/sweet_bow
+	name = "Sweet bow"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/head/head.dm:269]:
```suggestion
	name = "sweet bow"
```

File: modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm [hunk 29]
```diff
@@ -86,3 +86,30 @@
...
+	desc = "A neckpiece for a maid costume, it smells faintly of disappointment."
+	icon_state = "gags_maid_collar"
+	greyscale_config = /datum/greyscale_config/maid_costume/maid_collar
+	greyscale_config_worn = /datum/greyscale_config/maid_costume/maid_collar/worn
+	worn_icon = 'modular_skyrat/modules/GAGS/icons/maid_costume.dmi'
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/neck/face_scarf
+	name = "Face scarf"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm:101]:
```suggestion
	name = "face scarf"
```

File: modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm [hunk 30]
```diff
@@ -86,3 +86,30 @@
...
+	name = "Custom maid neck cover"
+	desc = "A neckpiece for a maid costume, it smells faintly of disappointment."
+	icon_state = "gags_maid_collar"
+	greyscale_config = /datum/greyscale_config/maid_costume/maid_collar
+	greyscale_config_worn = /datum/greyscale_config/maid_costume/maid_collar/worn
+	worn_icon = 'modular_skyrat/modules/GAGS/icons/maid_costume.dmi'
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/neck/face_scarf
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/_neck.dm:100]: Additionally, you'll want this to be a mask if you want it to be internals-ready.

File: modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm [hunk 31]
```diff
@@ -41,3 +41,40 @@
 /obj/item/clothing/neck/ranger_poncho/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/toggle_clothes, "ranger_poncho_t")
+
+/obj/item/clothing/neck/cape
+	name = "Cape"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm:46]:
```suggestion
	name = "long cape"
```

Comment (Nerev4r) [reply modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm:46]: Just a suggestion so it feels a little less generic as a name.

File: modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm [hunk 32]
```diff
@@ -41,3 +41,40 @@
...
+	body_parts_covered = CHEST|ARMS
+	flags_1 = IS_PLAYER_COLORABLE_1
+	heat_protection = CHEST
+
+/obj/item/clothing/neck/cape/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/toggle_clothes, "gags_cape_t")
+
+/obj/item/clothing/neck/robes
+	name = "Robes"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm:61]:
```suggestion
	name = "robe cape"
```

File: modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm [hunk 33]
```diff
@@ -41,3 +41,40 @@
...
+	desc = "comfortable robes, which lie softly along your back."
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm:62]:
```suggestion
	desc = "A comfortable northern-style cape, draped down your back and held around your neck with a brooch. Reminds you of a sort of robe."
```

File: modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm [hunk 34]
```diff
@@ -41,3 +41,40 @@
...
+	icon_state = "gags_robes"
+	greyscale_config = /datum/greyscale_config/cloak/robes
+	greyscale_config_worn = /datum/greyscale_config/cloak/robes/worn
+	greyscale_colors = "abd0e0"
+	body_parts_covered = CHEST|ARMS
+	flags_1 = IS_PLAYER_COLORABLE_1
+	heat_protection = CHEST
+
+/obj/item/clothing/neck/wide_cape
+	name = "Wide cape"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm:72]:
```suggestion
	name = "wide cape"
```

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm [hunk 35]
```diff
@@ -206,3 +206,14 @@
 	greyscale_config = /datum/greyscale_config/tailored_short_jacket
 	greyscale_config_worn = /datum/greyscale_config/tailored_short_jacket/worn
 	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#bf9f78#8c8c8c#8c8c8c#bf9f78#8c8c8c"
+
+/obj/item/clothing/suit/warm_coat
+	name = "Warm coat"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/coats.dm:211]:
```suggestion
	name = "warm coat"
```

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm [hunk 36]
```diff
@@ -206,3 +206,14 @@
...
+	desc = "A long insulated coat with fur, it looks quite comfortable."
+	icon_state = "gags_warm_coat"
+	greyscale_config = /datum/greyscale_config/warm_coat
+	greyscale_config_worn = /datum/greyscale_config/warm_coat/worn
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	body_parts_covered = CHEST|ARMS
+	flags_1 = IS_PLAYER_COLORABLE_1
+	heat_protection = CHEST
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/coats.dm:219]:
```suggestion
	cold_protection = CHEST|GROIN|ARMS
	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```

File: modular_skyrat/modules/customization/modules/clothing/suits/coats.dm [hunk 37]
```diff
@@ -206,3 +206,14 @@
...
 	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#bf9f78#8c8c8c#8c8c8c#bf9f78#8c8c8c"
+
+/obj/item/clothing/suit/warm_coat
+	name = "Warm coat"
+	desc = "A long insulated coat with fur, it looks quite comfortable."
+	icon_state = "gags_warm_coat"
+	greyscale_config = /datum/greyscale_config/warm_coat
+	greyscale_config_worn = /datum/greyscale_config/warm_coat/worn
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	body_parts_covered = CHEST|ARMS
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/coats.dm:217]:
```suggestion
	body_parts_covered = CHEST|GROIN|ARMS
```

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 38]
```diff
@@ -245,9 +245,6 @@
 	icon_state = "kuban_cossak"
 	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
 
-/obj/item/clothing/suit/costume/ghost_sheet
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:248]: Also this.

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 39]
```diff
@@ -306,6 +303,28 @@
 	greyscale_config_worn_vox = /datum/greyscale_config/apron/worn/oldvox
 	flags_1 = IS_PLAYER_COLORABLE_1
 
+/obj/item/clothing/suit/warm_sweater
+	name = "Warm sweater"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm]:
```suggestion
	name = "warm sweater"
```

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 40]
```diff
@@ -306,6 +303,28 @@
...
+	desc = "A pretty sweater, it looks quite comfortable and warm."
+	icon_state = "gags_warm_sweater"
+	greyscale_config = /datum/greyscale_config/warm_sweater
+	greyscale_config_worn = /datum/greyscale_config/warm_sweater/worn
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	body_parts_covered = CHEST
+	flags_1 = IS_PLAYER_COLORABLE_1
+	heat_protection = CHEST
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm]:
```suggestion
	cold_protection = CHEST|GROIN|ARMS
	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 41]
```diff
@@ -306,6 +303,28 @@
...
 	flags_1 = IS_PLAYER_COLORABLE_1
 
+/obj/item/clothing/suit/warm_sweater
+	name = "Warm sweater"
+	desc = "A pretty sweater, it looks quite comfortable and warm."
+	icon_state = "gags_warm_sweater"
+	greyscale_config = /datum/greyscale_config/warm_sweater
+	greyscale_config_worn = /datum/greyscale_config/warm_sweater/worn
+	greyscale_colors = "#8c8c8c#8c8c8c#8c8c8c#8c8c8c"
+	body_parts_covered = CHEST
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm]:
```suggestion
	body_parts_covered = CHEST|GROIN|ARMS
```

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 42]
```diff
@@ -306,6 +303,28 @@
...
+	heat_protection = CHEST
+
+/obj/item/clothing/suit/heart_sweater
+	name = "Heart sweater"
+	desc = "A pretty sweater, it looks quite comfortable and warm. It also has a heart pattern, how cute."
+	icon_state = "gags_heart_sweater"
+	greyscale_config = /datum/greyscale_config/heart_sweater
+	greyscale_config_worn = /datum/greyscale_config/heart_sweater/worn
+	greyscale_colors = "#8c8c8c#8c8c8c"
+	body_parts_covered = CHEST
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:325]:
```suggestion
	body_parts_covered = CHEST|GROIN|ARMS
```

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 43]
```diff
@@ -306,6 +303,28 @@
...
+	flags_1 = IS_PLAYER_COLORABLE_1
+	heat_protection = CHEST
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:327]:
```suggestion
	cold_protection = CHEST|GROIN|ARMS
	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```

**Review by Nerev4r — COMMENTED** [review]
nerev4r 4maintr 2024r

**Comment by ghost** [comment]
Few things here. Just some important comments to start:
- _The sprites are really nice._ Like, legitimately, the capes especially have some really cool designs, and the other items like the face scarf just seem like they fill a gap I didn't realize was there. I've not reviewed them properly but just the glance through IconDiffBot is amazing.
- There's a lot of big work and little tweaks that are gonna be asked and be done, BUT, please fill out the PR body. Not just for Maintainers to see what this is meant to be, but to help yourself keep track of what's done and what needs done. Github even has a checklist function that you can easily check off stuff when it's completed, fullly implemented, etc.

For the actual work that I'll recommend getting done, this is the summary:
- Up on TG I've recently tried to push for some GAGS sorting and overhauls. I'm trying to ~~shoehorn~~ kindly suggest implementing these changes on Skyrat, to ease everybody's use of the system... one of the biggest things is a change from the two-json setup you've got here, to a one-json setup. I'll explain it in proper detail separately, but it'll be the majority of the changes. **Technically** it's not required, and I don't have the authority to make it required, but it makes troubleshooting the icons easier in-game, makes it easier to read the code, yada-yada.
Most of this one is detailed under code\datums\greyscale\README.md. It's a lot, and probably won't make sense _alone_, but I do recommend reading it if you're able. It covers basically everything I'll list in the comment(s) below, so if you're confident, stubborn, don't want to listen to me, whatever you choose, that's the source on what I'm writing.

**Comment by ghost** [comment]
### Step 1: Reorganizing the .dmis
To switch to single-json, the .dmi's will need a lot of reorganizing. Namely, splitting. The key to cutting down a .json is adding a .dmi, specifically an `item_worn.dmi`. It ends up with a lot of cutting/pasting, but it'll be worth it I swear... Specifically what I'm asking be done is, luckily, writable in a checklist:
- [ ] For each `item.dmi`, create a file called `item_worn.dmi` 
image
`(Substitute _snout for _worn. The GAGS module is outdated and doesn't follow anything I'm suggesting. Yet.)`
- [ ] Move all Mob icons into the new `_worn` files
`Specifically, just cut out any icon intended to be shown worn by the character, and paste it into the _worn.dmi`
- [ ] Rename the icons between files to have matching `icon_states`. Look at how TG does their non-GAGS icons, that's essentially what we're doing here... this one's honestly just hard to understand until you see the pictures.
`This would be the item.dmi...`
image
`And this would be the item_worn.dmi`
image
The goal is that the icon_state matches up between the object and mob sprites.
- [ ] Every item that has a toggle state (`_t` or `_d`) will need these for the object icon now too. Just a consequence of moving to one-json. Luckily, you can 100% just copy-paste the normal object sprite, add _t to the end, and call it a day.
`Mob sprites with a toggle state...`
image
`Will need obj sprites with a toggle state. But luckily, the obj sprites don't ACTUALLY need to toggle.`
image
- [ ] For the time being, put an error sprite of some sort in here. A big placeholder pink square will do. Best to remove this later once the icon works though, since ~10 duplicate error icons in different files is what Skyrat is meant to be cutting back on and I'd rather we didn't contribute to it LOL
- [ ] Final check for icons: all `obj` icons should only have one directional state.
`Just something I noticed in a few of the .dmis, make sure that the dropped item's icon only has 1 direction in each state. Best saved for last, because when it's sorted you'll know exactly which icons are the object icons.`

**_Once the icon sorting is done, the configs will break. Horribly. See the next comment for fixing that._**

**Comment by ghost** [comment]
### Step 2: Adjusting the Greyscale Configs
These will be the least impacted by the icon file change. All they'll need is some minor tweaks, the /worn ones will need a new `icon_file` var and will also have their `json_config` var removed. Why remove the json config from the worn one? Inheritance! That'll be explained in Step 3.

For each `greyscale_config`...
- [ ] The base one needs 3 vars to be defined: `name`, `icon_file`, and `json_config`. This is pretty much unchanged.
image
- [ ] The worn one should be a subtype of the base; the same path, but ending in /worn. This will allow us to inherit the base's `json_config` (you already got this one down, except you were inheriting the icon file)
image
- [ ] The worn config needs only 2 vars defined: `name` and `icon_file`. For the `icon_file` you'll put the new .dmis, the _worn ones.
image

This is all you'll need of the configs, at least all I can predict you'll need. Might need to come back and tweak it later, depending on the items.

**Comment by ghost** [comment]
### Step 3: JSON Config changes

Now that the `mob` and `worn` icon_states are all the same (Step 1), the .json can use the same set of instructions to assemble their sprites. Thus, we can use a single .json!
On top of that, with the worn greyscale_config being a subtype (Step 2), it can inherit the json and have the only difference be that it uses the _worn icon file to build our sprite. Very, very helpful. Now we only have to write and maintain the 1 json.

.json files are fairly template-based. Not much I can do besides show the template... so, Per the README mentioned earlier:
```
{
	"full_icon_state_name": [
		{
			"type": "icon_state",
			"icon_state": "component_state_1",
			"blend_mode": "overlay",
			"color_ids": [ 1 ]
		},
		{
			"type": "icon_state",
			"icon_state": "component_state_2",
			"blend_mode": "overlay",
			"color_ids": [ 2 ]
		},
		{
			"type": "icon_state",
			"icon_state": "non_colorable_component_state",
			"blend_mode": "overlay",
		}
	]
}
```

But, since you've already got that down fairly well, really the only things to do here:
- [ ] Delete any `item_worn.json`. We don't need them any more.
- [ ] Go through each `item.json` and make sure the component_state matches up with the .dmi's new icon_states
- [ ] Try to set the full_icon_state_name to something unique and not a component_state. I don't think it does anything bad but now's not the time to test it.

After this one's done the PR should be ""testable"" again. I'll likely have missed something or there'll be an edge-case that came up, but that's just something that will happen when it happens.

Hopefully this all made sense. If not, ask @Nerev4r on discord, who will no doubt proceed to DM me. I'll hopefully be able to help with any issues, struggles, questions, whatever brought up.

**Technically.** This should be close to functioning as-is. **Technically** nothing I suggest/ask is required, and you could simply work with the old system and whatever else maintainers ask. No harm by me.
And if you do end up doing my suggestions, but it ends up too confusing: don't feel like you messed up or failed or are in trouble or anything sad. Don't worry! It's all just code for a game that runs on what probably could qualify as malware. No harm done.
